code/game/machinery/doors/door.dm

```diff
@@ -77,9 +77,14 @@
 		var/mob/living/M = AM
 		if(world.time - M.last_bumped <= 10) return	//Can bump-open one airlock per second. This is to prevent shock spam.
 		M.last_bumped = world.time
-		if(!M.restrained() && M.mob_size > MOB_SIZE_SMALL)
-			bumpopen(M)
-		return
+		if(!M.restrained())

```

Comment:
I just wanted to keep the previous behavior of the code in-tack while allowing simple_animals the ability to bump doors. Good question though.